### PR TITLE
Deps required for new urllib3 version + IP SAN support

### PR DESCRIPTION
**Pull request by caseydavenport**
Depends on:
- New libcalico release with https://github.com/projectcalico/libcalico/pull/153/files
- New felix release with https://github.com/projectcalico/felix/pull/1125

Currently, both are using my own forks.

### CODE REVIEW

File: calico_node/Dockerfile [hunk 1]
```diff
@@ -28,15 +28,17 @@ RUN apk --no-cache add wget ca-certificates libgcc && \
...
 # Install Python so libnetwork-plugin can run
 RUN apk --no-cache add python py-setuptools
 
 # Actually install libnetwork-plugin. A python dev environment (and git) is installed so libnetwork-plugin can be pip installed.
 # Finally the dev environment is removed. Doing this in one image layer minimized the final image size.
-RUN apk --no-cache add --virtual temp py-pip git gcc python-dev musl-dev && pip install git+https://github.com/projectcalico/libnetwork-plugin.git && \
-    pip install git+https://github.com/projectcalico/libcalico.git && apk del --purge temp
+RUN apk --no-cache add --virtual temp py-pip git gcc python-dev musl-dev && \
+    pip install git+https://github.com/projectcalico/libnetwork-plugin.git && \
+    pip install git+https://github.com/caseydavenport/libcalico.git@update-urllib && \
```
Comment (caseydavenport) [inline calico_node/Dockerfile]: This change is in PR: https://github.com/projectcalico/libcalico/pull/153

File: Makefile [hunk 2]
```diff
@@ -49,7 +49,7 @@ $(NODE_CONTAINER_BIN_DIR)/calico-felix:
 	-docker rm -f calico-felix
 	# Latest felix binaries are stored in automated builds of calico/felix.
 	# To get them, we pull that image, then copy the binaries out to our host
-	docker create --name calico-felix calico/felix:latest
+	docker create --name calico-felix caseydavenport/felix:latest
```
Comment (caseydavenport) [inline Makefile]: This change is in this pr: https://github.com/projectcalico/felix/pull/1125

**Comment by caseydavenport** [comment]
@robbrockbank Assigned to you since you reviewed the libcalico change as well.

**Comment by tomdee** [comment]
D'oh, this was causing the builds to fail so I checked in a fix for it already. I think the deps are in the wrong place in this PR anyway, so I'm closing it.

**Pull request closed without merging**